code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cs

```diff
@@ -0,0 +1,24 @@
+namespace biggest_of_n_numbers
+{
+    public class biggest_of_n_numbers
```

Comment:
Namespaces and classes should both use PascalCase, according to [this](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions).